### ISSUE TITLE
NTR: Fructose in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8858]
New term request Fructose in urine (segal_120814162858):

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000804
- label: Elevated urine fructose level
- parents: [Abnormal urine carbohydrate level (HP:0031979)](https://hpo.jax.org/app/browse/term/HP:0031979)
- pmid: PMID:7833921
- def: Amount of fructose in the urine above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8858
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal
see above
